src/Core/PluginCollection.php

```diff
@@ -104,7 +104,7 @@ protected function loadConfig()
      *
      * @param string $name The plugin name to locate a path for. Will return '' when a plugin cannot be found.
      * @return string
-     * @throws Cake\Core\Exception\MissingPluginException when a plugin path cannot be resolved.
```

Comment:
Need to add starting blackslash here to make it proper FQCN instead of changing it to just class name.